python/py-dockerpty/Portfile

```diff
@@ -22,14 +22,15 @@ checksums           rmd160  6d0604c549d301b92b98c2b25825ff04881e48cd \
                     sha256  9603174148835398767ee75ac763333321bf84f764de93891a5decd07aa58d08 \
                     size    19608
 
-python.versions     27 35 36
+python.versions     27 35 36 37 38
 
 if {${subport} ne ${name}} {
     depends_build-append \
                     port:py${python.version}-setuptools
 
     depends_lib-append \
-                    port:py${python.version}-docker
+        port:py${python.version}-docker \
+        port:py${python.version}-six
```

Comment:
why not keep these lines aligned with the rest of the Portfile as it was before?

Reply:
You're right @reneeotten, no reason but Emacs deciding otherwise and my not realising. Fixed.